### PR TITLE
Nova chance: react questions

### PR DESCRIPTION
**Pull request by GregWeil**
also removed the question model

### CODE REVIEW

File: src/presenters/question-item.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+import {getEditorUrl} from '../models/project';
+
+const iconHelp = 'https://cdn.glitch.com/f7224274-1330-4022-a8f2-8ae09dbd68a8%2Fask-for-help.svg?1494954687906';
+
+function truncateQuestion(question) {
+  const max = 140;
+  if (question.length > max) {
+    return question.substring(0, max - 3) + '…';
+  }
```
Comment (judeallred) [inline src/presenters/question-item.jsx]: the ellipsis is only one character, so i think this should be max -1

Comment (GregWeil) [reply src/presenters/question-item.jsx]: the old one was max - 5 + three dots :man_shrugging: but yeah that probably makes more sense

**Comment by GregWeil** [comment]
fixed new project pop breaking project models elsewhere, only max-1 truncation

**Comment by judeallred** [comment]
lgtm

**Pull request merged**